packages/user/src/model.ts

```diff
@@ -154,9 +153,10 @@ export class User implements ICurrentUser {
    */
   private _fetchUser(): void {
     // Read username, color and initials from URL
-    let name = decodeURIComponent(env.getParam('--username', ''));
-    let color = env.getParam('--usercolor', '');
-    let initials = decodeURIComponent(env.getParam('--initials', ''));
+    const urlParams = new URLSearchParams(location.search);
+    let name = urlParams.get('username') || '';
+    let color = urlParams.get('usercolor') || '';
+    let initials = urlParams.get('initials') || '';
```

Comment:
@hbcarlos  Small comment for the future, you should prefer the `??` nullish operator to the `||` or operator as much as possible.

The or operator may result in strange value if the parameter is a boolean or a number (as they may defined but false).

Reply:
Ouch, Thanks for the tip Frédéric!